plugins/module_utils/api.py

```diff
@@ -347,6 +347,11 @@ def run(self, ib_obj_type, ib_spec):
         # checks if the object type is member to normalize the attributes being passed
         if (ib_obj_type == NIOS_MEMBER):
             proposed_object = member_normalize(proposed_object)
+            # The WAPI API will never return the "create_token" field that causes a difference
+            # with the defaults of the module. To prevent this we remove the "create_token" option
+            # if it has not been set to true.
+            if(proposed_object.get("create_token") is not True):
+                proposed_object.pop("create_token")
 
```

Comment:
Hi @matthewdennett , These lines creating an error with KeyError: create_token. 
**Suggested Fix**
Please remove only these lines 
```
if(proposed_object.get("create_token") is not True):
    proposed_object.pop("create_token")
``` 
Then this will fix the member idempotency, for me that was the case